docs/installing.md

```diff
@@ -12,40 +12,35 @@ To install on Linux or MacOs run
 ```sh
 bash < <(curl -sSLf https://kubeshop.github.io/testkube/install.sh )
 ```
-Alternative install 
+Alternative install:
+
 1. Download binary of your choice
 2. Upack it (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz)
-3. Move it to location in the path for example `mv  testkube_0.6.5_Linux_arm64/kubectl-testkube /usr/local/bin/kubectl-testkube`
+3. Move it to location in the PATH for example `mv  testkube_0.6.5_Linux_arm64/kubectl-testkube /usr/local/bin/kubectl-testkube`
 
-For Windows download desired binary from https://github.com/kubeshop/testkube/releases, unpack the binary and add it to `%PATH%`. 
+For Windows download desired binary from [here](https://github.com/kubeshop/testkube/releases), unpack the binary and add it to `%PATH%`. 
 
 We have plans to build installers for most popular OS and system distros.
 
 ## Install `testkube` components in your cluster
 
-The testkube kubectl plugin provides an install command to install testkube in your cluster. Internally 
-this uses Helm and so you will need to have recent `helm` command installed on your system.
+The testkube kubectl plugin provides an install command to install testkube in your cluster. 
 
 Run 
 ```shell
 kubectl testkube install
 ```
 
-You should have everything installed 🏅
-
-By default testkube is installed in `testkube` namespace but you can change it in manual install if you want.
-
-If you want testkube to provide the endpoint for the kubest dashboard use `kubectl testkube install -i` with the `-i` or `--ingress` option, it will setup a ingress-nginx controller for you in a managed cluster(for baremetal clusters this should be set up manually before installing testkube).
-## Uninstall `testkube`
-
-You can uninstall TestKube using uninstall command integrated into testkube plugin. 
+Above command will install following components in your Kubernetes cluster:
```

Comment:
Maybe here you can provide an example how the components should look with `kubectl get all -n testkube`.

Reply:
take another look i extended it